templates/zerver/help/configure-who-can-edit-topics.md

```diff
@@ -2,17 +2,11 @@
 
 {!admin-only.md!}
 
-By default, anyone can edit any message topic. This is useful because it
-allows the community to keep conversations organized, even if some members
-are still learning how to use topics effectively.
-
-However, you can restrict the permission to edit topics of any message
-to other sets of roles:
-
-* Organization administrators
-* Organization administrators and moderators
-* Organization administrators and all members
-* Organization administrators and [full members](/help/restrict-permissions-of-new-members)
+By default, anyone can edit any message topic. This is useful because it allows
+the community to keep conversations organized, even if some members are still
+learning how to use topics effectively. However, you can
+[restrict](/help/roles-and-permissions) the ability to edit the topic of any
```

Comment:
I think we should have the link on the word "roles" , e.g. 

"However, you can restrict the ability the topic of any message to specific [roles](/help/roles-and-permissions)".

Reply:
Sure, I can redo it that way. I didn't like the "set of roles" formulation, but that phrasing works.